src/home_robot_hw/home_robot_hw/stretch_client/api.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import rospy
+
+from home_robot_hw.constants import ControlMode
+
+from .ros import StretchRosInterface
+from .modules.nav import StretchNavigationInterface
+from .modules.manip import StretchManipulationInterface
+from .modules.camera import StretchCameraInterface
+
+class StretchClient:
+    """Defines a ROS-based interface to the real Stretch robot. Collect observations and command the robot."""
+
+    def __init__(self, init_cameras: bool = True, init_node: bool = False):
+        # Ros
+        if init_node:
+            rospy.init_node("stretch_user_client")
+        self._ros_client = StretchRosInterface()
```

Comment:
No, I made `StretchRosInterface` as an access point for all publishers and subscribers, instead of having them spread out across the camera, nav, and manip modules. The implementation is in `stretch_client/ros.py`

Reply:
yes agree, we should delete the old ROS interface too

Reply:
I can remove it in the next PR if no one is using it.
I hoped to keep this PR a pure addition.

Reply:
i think we may be using it but everyone should be forced to remove it asap